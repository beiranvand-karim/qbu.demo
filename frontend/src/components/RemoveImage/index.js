import React from "react"

const RemoveImage = ({removeImageFn, image}) => {
   if (image) {
      return <button onClick={removeImageFn}>remove image</button>
   }
   return null
};
export default RemoveImage