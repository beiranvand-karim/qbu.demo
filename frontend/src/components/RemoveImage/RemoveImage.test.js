/* global describe, it, expect */
import React from "react"
import RemoveImage from "./"
import {shallow} from "enzyme"
import renderer from 'react-test-renderer'
import {MemoryRouter} from "react-router-dom"

describe('<RemoveImage />', () => {

   it('should renderer without crashing', () => {
      shallow(<RemoveImage />)
   });
   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><RemoveImage /></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});