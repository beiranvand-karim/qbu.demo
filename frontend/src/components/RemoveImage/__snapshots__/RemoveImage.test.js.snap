// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RemoveImage /> should matches the snapshot 1`] = `null`;
